src/Provider/AlgoliaPlaces/AlgoliaPlaces.php

```diff
@@ -110,7 +126,35 @@ protected function getRequest(string $url): RequestInterface
             $this->buildData()
         );
     }
+    /**
+     * Send request and return contents. If content is empty, an exception will be thrown.
+     *
+     * @param RequestInterface $request
+     *
+     * @return string
+     *
+     * @throws InvalidServerResponse
+     */
+    protected function getParsedResponse(RequestInterface $request): string
```

Comment:
still weird for me to have to rewrite this method

Reply:
I agree,
Feel free to try it yourself: 
the tests at home passed alright (php7.2 and php7.0) without the getParsedResponse function, but Travis throw me an error if it is not there.